README.md

```diff
@@ -1,6 +1,6 @@
 # User Management UI Demo
-Applicazione ASP.NET Core 5 che dimostra come usare lo `UserManager<TUser>` per assegnare ruoli e altro tipo di claim agli utenti.
+Applicazione ASP.NET 8 che dimostra come usare lo `UserManager<TUser>` per assegnare ruoli e altro tipo di claim agli utenti.
```

Comment:
No, il nome corretto è ASP.NET Core 8.
https://learn.microsoft.com/it-it/aspnet/core/release-notes/aspnetcore-8.0?view=aspnetcore-9.0

Reply:
Fixato il nome corretto (ASP.NET Core 8) in questo commit: https://github.com/BrightSoul/UserManagementUiDemo/pull/3/commits/e0e75ade3dfe02bd49fc65d2977156c2f0b33b99